test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -188,3 +191,100 @@ var _ = SIGDescribe("[Feature:GPUDevicePlugin]", func() {
 		testNvidiaGPUs(f)
 	})
 })
+
+func testNvidiaGPUsJob(f *framework.Framework) {
+	_ = SetupNVIDIAGPUNode(f, false)
+	completions := int32(5)
```

Comment:
Maybe add a simple comment on why we need to run jo for 5 times? I guess it is to make sure the job runs long enough to experience node recreation?

Reply:
Thanks, updated.